docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+  repetition. In many cases an internal event or call may be repeated dozens or
+  even hundreds of times, leading to redundancy in the captured logs. A
+  developer or system administrator may then wish to reduce the logging of
+  similar events by lowering the sampling ratio - for example, if the ratio is
+  reduced to 0.5 (i.e. 50% sampling ratio), then a single event that occurs 10
+  times would only be logged 5 times in the resulting trace data.
+
+### Preprocessing child workflow configuration
+
+The following two sections are for configuring a [child workflow] as part of the
+ingest workflow run by Enduro. A child workflow is a concept borrowed from
+Enduro's [workflow engine], [Temporal] where one workflow spawns an ancillary
+workflow. We use this model in Enduro to keep custom activities with
+organization-specific business rules or policy requirements separated from the
+underlying general Enduro code. For more information, see:
+
+* [Preprocessing child workflow](../dev-manual/preprocessing.md)
+
+**Default values**:
+
+```toml
+[preprocessing]
+enabled = false
+extract = false
+sharedPath = "/home/enduro/preprocessing"
+```
+
+* `enabled`: Boolean value, set to `false` by default. To enable and configure a
+  custom child workflow, change to `true`.
+* `extract`: Boolean value, set to `false` by default. This setting determines
+  whether SIP extraction happens as part of the child workflow or not. When set
+  to false, SIP extraction will occur at the beginning of the workflow. In some
+  cases, you may wish to design custom child workflow activities that perform
+  operatons on the SIP before it is extracted - for example, calculating a
+  checksum of the zipped package to check for prior duplicate ingests before
+  proceeding. When this value is set to `true` Enduro will skip the extraction
+  task at the beginning of the ingest workflow, allowing you to define when and
+  how extraction occurs in the child workflow.
+* `sharedPath`: The absolute path to the directory that Enduro should use to
+  share the SIP between the primary ingest workflow and the configured
+  preprocessing child workflow. This path is required when `enabled` is set to
+  true.
+
+#### Temporal workflow configuration for preprocessing workflow
+
+These settings ensure that Enduro can connect with [Temporal] and trigger the
+configured [child workflow].
+
+**Default values**:
+
+```toml
+[preprocessing.temporal]
+namespace = "default"
+taskQueue = "preprocessing"
+workflowName = "preprocessing"
+```
+
+* `namespace`: An internal namespace label for the temporal workflows. This only
+  needs to be changed if you expect to be running multiple different workflows
+  running at once, so they don't clash.
+* `taskQueue`: In Temporal, a [Task Queue] is  a lightweight, dynamically
+  allocated queue that one or more workers can poll for tasks. Temporal supports
+  all kinds of parallelization for a distributed application architecture, and
```

Comment:
Maybe "Temporal support *many* kinds of parallelization"?  I don't think supporting *all* kinds of parallelization is possible. 😆

Reply:
Fixed now - there were multiple occurances of this apparently and I didn't address them all.